content/blog/2019/knative-activator-adapter/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Mixer out-of-process adapter for Knative
+subtitle: Demonstrates a Mixer out-of-process adapter which implements the Knative scale-from-zero logic
+description: Demonstrates a Mixer out-of-process adapter which implements the Knative scale-from-zero logic.
+publishdate: 2019-07-18
+attribution: Idan Zach (IBM)
+keywords: [mixer,adapter,knative,scale-from-zero]
+---
+
+This post demonstrates how you can leverage [Mixer](https://istio.io/faq/mixer/) to push application logic
+into Istio. It describes a Mixer adapter which implements the [Knative](https://knative.dev/) scale-from-zero logic
+with a very simple code and similar performance to the original implementation.
+
+## Knative serving
+
+[Knative Serving](https://knative.dev/docs/serving/) builds on [Kubernetes](https://kubernetes.io/) to support deploying
+and serving of serverless applications. A core capability of serverless platforms is scale-to-zero
+functionality which reduces resource usage and cost of inactive workloads. However, this requires a new mechanism - when
```

Comment:
remove `However, This...` -> "A new mechanism is required to scale to zero when an idle application receives a new request."